src/index/IndexImpl.cpp

```diff
@@ -689,10 +689,9 @@ void IndexImpl::createFromOnDiskIndex(const string& onDiskBase) {
 
   if (_usePatterns) {
     PatternCreator::readPatternsFromFile(
-        _onDiskBase + ".index.patterns",
-        _fullHasPredicateMultiplicityPredicates,
-        _fullHasPredicateMultiplicityEntities, _fullHasPredicateSize, _patterns,
-        _hasPattern);
+        _onDiskBase + ".index.patterns", _avgNumDistinctSubjectsPerPredicate,
+        _avgNumDistinctPredicatesPerSubject, _numDistinctSubjectPredicatePairs,
+        _patterns, _hasPattern);
```

Comment:
Do I understand correctly that these are the same quantities as before, just with more appropriate names?